src/ripple/rpc/handlers/AccountCurrenciesHandler.cpp

```diff
@@ -52,6 +53,14 @@ Json::Value doAccountCurrencies (RPC::Context& context)
     if (auto jvAccepted = RPC::accountFromString (accountID, strIdent, bStrict))
         return jvAccepted;
 
+    auto const sleAccepted = ledger->read(keylet::account(accountID));
+    if (! sleAccepted)
+    {
+        result[jss::account] = context.app.accountIDCache().toBase58 (accountID);
```

Comment:
I'm not super familiar with rpc results, but it seems like we could get by without this since the account is in the request.
It looks like `account_info` includes `"account"` with `rpcACT_NOT_FOUND` but other places do not.